src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_node_explicit_storage.h

```diff
@@ -11,6 +11,13 @@
 #ifndef SIMPLEX_TREE_SIMPLEX_TREE_NODE_EXPLICIT_STORAGE_H_
 #define SIMPLEX_TREE_SIMPLEX_TREE_NODE_EXPLICIT_STORAGE_H_
 
+// Empty base optimization for MSVC - https://learn.microsoft.com/en-us/cpp/cpp/empty-bases
+#if _MSC_VER && !__INTEL_COMPILER
```

Comment:
Any reference for the `!__INTEL_COMPILER`?

Reply:
It was mentionned in [this stackoverflow](https://stackoverflow.com/questions/5850358/is-there-a-preprocessor-define-that-is-defined-if-the-compiler-is-msvc), but yes this may not be the case for us. I remove it.